GATEWAY/src/providers/ble.service.ts

```diff
@@ -84,6 +84,7 @@ export class BleService {
                   this.ble.enable().then( res => {
                     this.checkLocation();
                     resolve();
+                    console.log(err);
```

Comment:
Should be removed, trying to keep number of console logs as small as possible, and also it will never run as the promise resolves before it.